src/components/mapgl/mapgl.component.ts

```diff
@@ -917,6 +926,15 @@ export class MapglComponent implements OnInit, AfterViewInit, OnChanges {
       }
     });
 
+    this.redrawSource.subscribe(sd => {
```

Comment:
Question : This redrawSource replace redrawTile ?

Reply:
Yes, I removed redrawTile completely